ansible_image_validation/validate-vm-images.yaml

```diff
@@ -10,7 +10,8 @@
   remote_user: "{{ lookup('env', 'ADMIN_USER_NAME') }}"
   become: true
   vars:
-    ansible_become_pass: "{{ lookup('env', 'ADMIN_PASSWORD') }}"
+    ansible_become_pass: "{{ lookup('env', 'ADMIN_USER_NAME') }}"
+    is_rhui_package_present: true
```

Comment:
Do we have a corresponding variable setup in our ADO pipeline that changes this to false, in case of BYOS images? -> no, for BYOS we don't check for client package

```
- name: Check for Rhui client package in the image
  when: license_type != 'byos'
```

Reply:
changed the default value to false

Reply:
Is this variable ever updated to a value other than the default? 
per-region-validation.yml needs to run for when the value is true (PAYG)
and not run when value is false (BYOS)

Where is this getting changed?

Reply:
Got it. But this won't catch the bug that led to the issue. If the client package embedded in sap image is 0, it would treat it as is_rhui_client_package_present false and not run regional validation and pass the test, isn't it?